src/test/scala/mesosphere/marathon/raml/PodStatusConversionTest.scala

```diff
@@ -407,6 +407,30 @@ class PodStatusConversionTest extends UnitTest {
           "persistent#volume#5425cbaa-8fd3-45f0-afa4-74ef4fcc594b")))
     }
 
+    "persistent pod launched, received RUNNING status from Mesos, task endpoint health looks great" in {
```

Comment:
how is this test related to GPUs?

Reply:
It's something from another changelist :) good catch!